src/status.h

```diff
@@ -268,6 +268,39 @@ typedef enum {
     JSConsumerCreateFilterSubjectMismatchErr = 10131,   ///< Consumer create request did not match filtered subject from create subject
     JSConsumerCreateDurableAndNameMismatchErr = 10132,  ///< Consumer Durable and Name have to be equal if both are provided
     JSReplicasCountCannotBeNegativeErr = 10133,         ///< Replicas count cannot be negative
+    JSConsumerReplicasShouldMatchStreamErr = 10134,     ///< Consumer config replicas must match interest retention stream's replicas
```

Comment:
Added all of the missing codes from the server, including the 2 needed here for testing